browser/js/content.js

```diff
@@ -64,52 +66,69 @@ function renderChatContact(el, user) {
   const keybaseUsername = user.services["keybase"];
   let queryStatus, iconSrc;
   if (keybaseUsername) {
-    queryStatus = `<a class="keybase-user" href="https://keybase.io/${keybaseUsername}" target="_blank"><span>${keybaseUsername}</span></a> on Keybase`;
-    iconSrc = `<img class="keybase-icon" src="https://keybase.io/${keybaseUsername}/picture" />`;
+    queryStatus = bel`
+      <small>
+        <a class="keybase-user" href="https://keybase.io/${keybaseUsername}" target="_blank">
+          <span>${keybaseUsername}</span>
+        </a> on Keybase
+      </small>
+    `;
+    iconSrc = bel`<img class="keybase-icon" src="https://keybase.io/${keybaseUsername}/picture" />`;
   } else {
-    queryStatus = "Searching...";
+    queryStatus = bel`<small>Searching...</small>`;
     if (keybaseUsername === null) {
-      queryStatus = "(Not yet on Keybase)";
+      queryStatus = bel`<small>(Not yet on Keybase)</small>`;
     }
-    iconSrc = `
+    iconSrc = bel`
       <img class="keybase-icon"
            src="${asset("images/icon-placeholder-avatar-32.png")}"
            srcset="${asset("images/icon-placeholder-avatar-32@2x.png")} 2x, ${asset("images/icon-placeholder-avatar-32@3x.png")} 3x"
            />
     `;
   }
-  el.innerHTML = `
-    <div>${iconSrc}</div>
-    Encrypt to <a class="keybase-user ${user.origin}" href="${user.href()}" target="_blank">${user.display()}</a>
-    <small>${queryStatus}</small>
-  `;
+  morphdom(el, bel`
+    <div class="keybase-contact">
+      <div>${iconSrc}</div>
+      Encrypt to <a class="keybase-user ${user.origin}" href="${user.href()}" target="_blank">${user.display()}</a>
```

Comment:
Out of curiosity, does the `safeHTML` filtering interact with `bel`? I'm wondering if entities will be interpreted or rendered as text.